src/js/components/Widgets/OfficeNameText.jsx

```diff
@@ -24,34 +24,34 @@ export default class OfficeNameText extends Component {
   render () {
     renderLog(__filename);
     let nameText = "";
-    const { contest_office_name, political_party } = this.props;
-    if (political_party === undefined) {
+    const { contestOfficeName, politicalParty } = this.props;
+    if (politicalParty === undefined) {
       nameText = (
         <span className="no-political-party">
           <span>Candidate for </span>
-          { this.props.office_link ? (
-            <Link to={this.props.office_link}>
-              <span className="candidate-card-main__office">{ contest_office_name }</span>
+          { this.props.officeLink ? (
```

Comment:
Is there any reason not to include `officeLink` in the deconstruction you do above on line 27?

Reply:
No particular reason to not include offLink in the deconstruction. It wasn't included previously so I wasn't sure to include it in the refactor.

Reply:
Could you please include `officeLink` in the deconstruction and update your pull request? Thank you!
Now I see your note saying you were able to do some testing. Sorry I missed that the first time I looked at this pull request.

Reply:
I'll resubmit to include offLink in the deconstruction. 

I was able to test a few items in the browser's react extension to see the if the props were being based down correctly. Also, visually see if the party name, office, and link are being rendered. 

I am having some trouble finding OrganizationVoterGuideCandidateItem.jsx and VoterPositionItem.jsx. Any insight on where they are in the app?

Reply:
WebApp/src/js/components/VoterGuide/OrganizationVoterGuideCandidateItem.jsx

...gets called by:
WebApp/src/js/components/VoterGuide/OrganizationVoterGuideCandidateList.jsx
(which is called by: WebApp/src/js/routes/VoterGuide/OrganizationVoterGuideOffice.jsx)
and:
WebApp/src/js/routes/VoterGuide/OrganizationVoterGuideCandidate.jsx

You can look up the paths in WebApp in this files:
WebApp/src/js/Root.jsx

Examples are:
OrganizationVoterGuideCandidate: http://localhost:3000/candidate/wv02cand40137/btvg/wv01org14

OrganizationVoterGuideOffice: http://localhost:3000/office/wv02off19951/btvg/wv01org14

I can show you if you would like to join the hangout (you can ask me written questions via Slack, and I can respond verbally.)

Reply:
Thanks for showing me where the paths are. It'll help in the future on finding where things are. I'll send more messages through Slack next time. 

I also included officeLink in the destructuring  and things looked okay on my end.